js/components/input.jsx

```diff
@@ -1,5 +1,9 @@
 var React = require('react');
-var AppDispatcher = require('../dispatcher/app_dispatcher');
+var Actions = require('../actions/actions');
+var KeyCodes = {
+  enter: 13
+};
+var CommandCharacter = ":";
```

Comment:
So the reason I like : is because the next one I'm adding is s/old/new. Anyway, the solution here is to make it configurable. There should be a UserConfigStore and a way to set the command character attribute. https://github.com/rorysaur/quack/issues/17